### DISCUSSION TITLE
AMBA Deploy Azure Monitor Baseline Alerts - Policy Inclusion

### DISCUSSION THREAD

**Discussion by MilesCameron-DMs** [Azure/Enterprise-Scale#1481] [General]
Hi there,

The following page seems to suggest that the Deploy Azure Monitor Baseline Alerts definitions are included in the Azure landing zone definitions: [Deploy Azure Monitor Baseline Alerts](https://github.com/Azure/Enterprise-Scale/wiki/ALZ-Policies#what-policy-definitions-are-assigned-within-the-azure-landing-zones-custom--built-in) although this may suggest it simply assigns them if they exist - its not clear.

I have trawled through the Enterprise-Scale repo and cannot see the AMBA definitions - we consume the ALZ-Bicep repo downstream using GitHub Actions.

The literature is a bit confusing around this - do we need to consume the AMBA repo separately to receive the definitions?

Apologies if i missed something but appears to me that its only included in the portal accelerator at the moment and we must either clone the repo separately or perhaps merge this with the ALZ-Bicep repo.

Any guidance based on experience would be much appreciated as we are keen to use the baselines.

**Comment by jtracey93** [comment]
Hey @MilesCameron-DMs,

We are working on adding them to the Bicep & TF implementations as we speak, so they will appear like you are used to seeing them in those implementations.

However, today they are only in the portal experience. So if you need them right now, you can do so by following: https://azure.github.io/azure-monitor-baseline-alerts/patterns/alz/deploy/Introduction-to-deploying-the-ALZ-Pattern/

Tagging @arjenhuitema for visibility and for his review if we should updated the doc mentioned by @MilesCameron-DMs to make this clearer?

**Comment by MilesCameron-DMs** [comment]
Thanks @jtracey93 

I am just glad that i wasnt missing something.

Some clarity in the wiki would be much appreciated - as simple as saying its included in the portal experience but if you want to add it via these other methods then you must do this < insert high level steps and links >
